finishline

```diff
@@ -143,6 +152,13 @@ def parse_arguments():
     if not args.since:
         args.since = default_since
 
+    # Allow for service account password from environment variable
+    env_passwd = os.getenv('FINISHLINE_SERVICE_ACCOUNT_PASSWORD')
+    if args.service_account_password is None and env_passwd is not None:
+         args.service_account_password = env_passwd
+    if args.service_account_password is not None:
+        print('WARNING: service account password in use')
```

Comment:
Last bit.  May want to print to `stderr` here so that the output doesn't jack up the markdown output on `stdout`.

I think `print(..., file=sys.stderr)` should do it.

Reply:
Pushed in a fixup commit.